tests/routing_table/test_routing_table_utils.py

```diff
@@ -306,6 +307,13 @@ def test_expand_entry():
         RoutingTableEntry({Routes.north}, 0x1, 0xffffffff),
     ]
 
+    # The same, but ensuring that the source information isn't lost.
+    entry = RTE({Routes.north}, 0x0, 0xfffffffe, {Routes.south})
+    assert list(expand_entry(entry)) == [
+        RoutingTableEntry({Routes.north}, 0x0, 0xffffffff, {Routes.south}),
```

Comment:
Super-high nitpick: If you're using the "RTE" alias above I'd use it everywhere since it is a little confusing otherwise.